example_test.go

```diff
@@ -271,3 +272,21 @@ func ExampleUsage() {
 	//   THREE string
 	//     	third parameter
 }
+
+func ExampleUnexportedField(t *testing.T) {
+	conf := struct {
+		Database struct {
+			Host        string `yaml:"host" env:"DB_HOST" env-description:"Database host"`
+			Port        string `yaml:"port" env:"DB_PORT" env-description:"Database port"`
+		} `yaml:"database"`
+		server struct {
+			Host string `yaml:"host" env:"SRV_HOST,HOST" env-description:"Server host" env-default:"localhost"`
+			Port string `yaml:"port" env:"SRV_PORT,PORT" env-description:"Server port" env-default:"8080"`
+		} `yaml:"server"`
+	}{}
+
+	if err := cleanenv.ReadConfig("example/simple_config/config.yml", &conf); err != nil {
+		t.Fatal(err)
+	}
+	t.Logf("%+v", conf)
+}
```

Comment:
Please add a newline at the file end.

Reply:
I have a newline on my local editor but it seems not pushing to remote branch